src/view/com/post-thread/PostThread.tsx

```diff
@@ -255,6 +255,28 @@ export function PostThread({uri}: {uri: string | undefined}) {
     randomCache,
   ])
 
+  const [collapsedCIDs, setCollapsedCIDs] = React.useState(new Set())
+  const descendantsOfCollapsedCIDs = React.useMemo(() => {
+    const set = new Set<string>()
+    if (!skeleton) return set
+
+    for (let val of skeleton.replies) {
+      if (!isThreadPost(val)) continue
+      if (collapsedCIDs.has(val.post.cid) || set.has(val.post.cid)) {
+        val.replies?.forEach(r => {
+          isThreadPost(r) && set.add(r.post.cid)
+        })
+      }
+    }
```

Comment:
This traverses the tree of posts any time a post is collapsed, which may not be efficient enough in production